crates/rust-analyzer/src/handlers.rs

```diff
@@ -762,6 +762,7 @@ fn handle_fixes(
             group: None,
             kind: Some(CodeActionKind::QUICKFIX),
             edit: Some(edit),
+            is_preferred: Some(true),
```

Comment:
We shouldn't mark **all** fixes as preferable. Like, if you are calling private method, the fix would be "make the method pub", but it might very well be that the right action is "don't call the method".

We should only apply `is_preferred` when it is unambigious that this single fix is required. 

For fixes from `cargo check`, I think that would be "is_machine_applicable". 

For our fixes, we need to add `is_definitive` / `is_anamigious` / `autofix` or some such to our internal fixit struct, and then use it on case-by-case bases, with the default being `false`.

Reply:
I will do that in a followup PR